cluster/juju/layers/kubernetes-master/layer.yaml

```diff
@@ -8,7 +8,7 @@ includes:
   - 'layer:metrics'
   - 'layer:nagios'
   - 'layer:cdk-service-kicker'
-  - 'layer:kubernetes-common'
```

Comment:
master and worker are no longer using kubernetes-common?

Reply:
They're inheriting it via kubernetes-master-worker-base. Do you want me to explicitly include it in kubernetes-master and kubernetes-worker too?

Reply:
-1 on being explicit, but it's a soft -1.  We don't explicitly list layers that get inherited from things like `layer:vaultlocker`.

That said, i could see how one might glance at the code and wonder how we're including common functions if the layer isn't listed, and then having to take a minute to figure out where they must be coming from...

So this comment is basically worthless. I'm confident you'll do the right thing.

Reply:
-1 on being explicit as well, since the point of the base layer is to avoid repeating things that are shared.

On a more general note, by philosophy is to be explicit about the include if the layer is explicitly referenced in the charm, unless it's being included by something that is an explicit base layer for a small set of related charms (i.e., not a general layer used by many unrelated charms).  If that makes any sense.